python/cuml/experimental/common/base.pyx

```diff
@@ -0,0 +1,548 @@
+            return original_func(*args, **kwargs)
+        elif device_type == DeviceType.host:
+            # check if the sklean model already set as attribute of the cuml
+            # estimator its presence should signify that CPU execution was
+            # used previously
+            if not hasattr(self, 'sk_model_'):
+                # import model in sklearn
+                if hasattr(self, 'sk_import_path_'):
+                    # if import path differs from the one of sklearn
+                    # look for sk_import_path_
+                    model_path = self.sk_import_path_
+                else:
+                    # import from similar path to the current estimator
+                    # class
+                    model_path = 'sklearn' + self.__class__.__module__[4:]
+                model_name = self.__class__.__name__
+                sk_model = getattr(import_module(model_path), model_name)
+                # initialize model
+                self.sk_model_ = sk_model()
+                # transfer params set during cuml estimator initialization
+                for param in self.get_param_names():
+                    self.sk_model_.__dict__[param] = self.__dict__[param]
+
+                # transfer attributes trained with cuml
+                for attribute in self.get_attributes_names():
+                    # check presence of attribute
+                    if hasattr(self, attribute):
+                        # get the cuml attribute
+                        cu_attr = self.__dict__[attribute]
+                        # if the cuml attribute is a CumlArrayDescriptorMeta
+                        if hasattr(cu_attr, 'get_input_value'):
+                            # extract the actual value from the
+                            # CumlArrayDescriptorMeta
+                            cu_attr_value = cu_attr.get_input_value()
+                            # check if descriptor is empty
+                            if cu_attr_value is not None:
+                                if cu_attr.input_type == 'cuml':
+                                    # transform cumlArray to numpy and set it
+                                    # as an attribute in the sklearn model
+                                    self.sk_model_.__dict__[attribute] = \
+                                        cu_attr_value.to_output('numpy')
+                                else:
+                                    # transfer all other types of attributes
+                                    # directly
+                                    self.sk_model_.__dict__[attribute] = \
+                                        cu_attr_value
+                        else:
+                            # transfer all other types of attributes directly
+                            self.sk_model_.__dict__[attribute] = cu_attr
+                    else:
+                        raise ValueError('Attribute could not get transfered')
```

Comment:
Adding the exact attribute that triggers this error will help with future debugging